### DISCUSSION TITLE
A side mounted pen rack?

### DISCUSSION THREAD

**Discussion by hujiba** [Sakati84/3DPrintColorizer#63] [Ideas]
A lot of the time taken by the print is moving the marker up and down. I have my Z axis speed as fast as I can on the Ender 3 V2, which i think is pretty slow. Would a side mounted pen rack be possible to reduce the travel distance?

**Comment by Sakati84** [comment]
I think not because than you would need to mount it to the moving build plate and that would be quite wobbly. But I´m open for ideas.
